packages/dashboard/src/components/DashboardFilters/DashboardFilters.tsx

```diff
@@ -1,205 +1,209 @@
-        return (
-          <div className={formGroupClass} key={`${filter.key}-filtersection-${filterIndex}`}>
-            {label && (
-              <label className='font-weight-bold mb-2' htmlFor={`filter-${filterIndex}`}>
-                {label}
-              </label>
-            )}
-            {filter.filterStyle === FILTER_STYLE.multiSelect ? (
-              <MultiSelect
-                label={label}
-                options={multiValues}
-                fieldName={filterIndex}
-                updateField={updateField}
-                selected={filter.active as string[]}
-                limit={filter.selectLimit || 5}
-                loading={loading}
-              />
-            ) : filter.filterStyle === FILTER_STYLE.nestedDropdown ? (
-              <NestedDropdown
-                activeGroup={(filter.queuedActive?.[0] || filter.active) as string}
-                activeSubGroup={_key ? filter.queuedActive?.[1] || filter.subGrouping?.active : activeSubGroupValue}
-                filterIndex={filterIndex}
-                options={_key ? getNestedDropdownOptions(apiFilterDropdowns[_key]) : nestedOptions}
-                listLabel={label}
-                handleSelectedItems={value => updateField(null, null, filterIndex, value)}
-                loading={loading}
-              />
-            ) : (
-              <>
-                <select
-                  id={`filter-${filterIndex}`}
-                  className={`cove-form-select ${DROPDOWN_STYLES}`}
-                  data-index='0'
-                  value={loading ? 'Loading...' : filter.queuedActive || filter.active}
-                  onChange={val => {
-                    handleOnChange(filterIndex, val.target.value)
-                  }}
-                  disabled={loading || isDisabled}
-                >
-                  {loading && <option value='Loading...'>Loading...</option>}
-                  {nullVal(filter) && (
-                    <option key={`select`} value=''>
-                      {filter.resetLabel || '- Select -'}
-                    </option>
-                  )}
-                  {values}
-                </select>
-                {loading && <Loader spinnerType={'text-secondary'} />}
-              </>
-            )}
-          </div>
-        )
-      })}
-      {showSubmit && (
-        <button
-          className='btn btn-primary mb-1'
-          onClick={applyFilters}
-          disabled={show.some(filterIndex => {
-            const emptyFilterValues = [undefined, '', '- Select -']
-            return (
-              emptyFilterValues.includes(sharedFilters[filterIndex].queuedActive) &&
-              emptyFilterValues.includes(sharedFilters[filterIndex].active)
-            )
-          })}
-        >
-          {applyFiltersButtonText || 'GO!'}
-        </button>
-      )}
-    </form>
-  )
-}
-
-export default DashboardFilters
+import React from 'react'
```

Comment:
for some reason the precommit hook is changing the line endings in this file. I think maybe at some point a PR went in that mixed the line endings. and the precommit hook is changing it to crlf like it should be?